{{cookiecutter.project_slug}}/.github/workflows/docs.yml

```
@@ -0,0 +1,42 @@
+name: Documentation
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
```

Going with the default name `build` can cause issues with things like [required status checks](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/collaborating-on-repositories-with-code-quality-features/troubleshooting-required-status-checks)
```suggestion
  docs:
```